docs/eloquent-models.txt

```diff
@@ -351,7 +351,7 @@ relation definition.
 
 .. code-block:: php
 
-   $book = Book::first();
+   $book = User::first()->books()->first();
```

Comment:
Correct, `Book::first()` sends a query to the `books` collection. Which is not expected for embeds.